ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactionQueryBuilder.java

```diff
@@ -419,15 +421,13 @@ private void defineColumns(StringBuilder query) {
     }
     List<FieldSchema> cols = sourceTab.getSd().getCols();
     boolean isFirst = true;
+    List<String> columnDescs = new ArrayList<String>();
```

Comment:
no need to provide type in the diamond operator,  from Java 1.7 adds type inference
````
List<String> columnDescs = new ArrayList<>()
````

Reply:
Fixed the code.